<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialDialog</class>
 <widget class="QDialog" name="MaterialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaterialDialog</string>
  </property>
  <widget class="QDoubleSpinBox" name="gSB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>32.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="sSBR">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Specular</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mrB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>移除</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="cSBR">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Main Texture</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sSBG">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="cSBG">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="rFR">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sSBB">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="cSBB">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="rFB">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>rF0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="rFG">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QPushButton" name="noB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nrB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>移除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="meoB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>530</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Metallic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="merB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>570</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>移除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="roB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>690</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Roughness</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rrB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>730</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>移除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gloss</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aoB">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>AO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="arB">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>移除</string>
   </property>
  </widget>
  <widget class="QLabel" name="mCav">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nCav">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="meCav">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rCav">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>680</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="aCav">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="mSB">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>520</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>490</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Metalness</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="bSB">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BumpScale</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sSB">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>680</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.001000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>650</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Smoothness</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
